packages/matrix/tests/messages.spec.ts

```diff
@@ -168,25 +173,70 @@ test.describe('Room messages', () => {
   test('can add a card that is over 65K to a message (i.e. split card into multiple matrix events)', async ({
     page,
   }) => {
-    const testCard = `${testHost}/mango-puppy`; // this is a 153KB card
+    const testCard = `${testHost}/big-card`; // this is a 153KB card
```

Comment:
Is the comment still valid for this file?

Reply:
I’ll update it

Reply:
actually this comment is still correct. the big card uses the same data as the previous mango-puppy card